types/chrome/index.d.ts

```diff
@@ -5847,6 +5893,480 @@ declare namespace chrome.system.storage {
+ * @since Chrome 30.
+ */
+declare namespace chrome.system.display {
+
+  export const DisplayPosition: {
+      TOP: 'top',
+      RIGHT: 'right',
+      BOTTOM: 'bottom',
+      LEFT: 'left'
+  };
+  export const MirrorMode: {
+      OFF: 'off',
+      NORMAL: 'normal',
+      MIXED: 'mixed'
+  };
+  export interface Bounds {
+      /**  The x-coordinate of the upper-left corner. */
+      left: number;
+      /**  The y-coordinate of the upper-left corner. */
+      top: number;
+      /** The width of the display in pixels. */
+      width: number;
+      /** The height of the display in pixels. */
+      height: number;
+  }
+
+  export interface Insets {
+      /** The x-axis distance from the left bound. */
+      left: number;
+      /** The y-axis distance from the top bound. */
+      top: number;
+      /** The x-axis distance from the right bound. */
+      right: number;
+      /** The y-axis distance from the bottom bound. */
+      bottom: number;
+  }
+
+  /**
+   * @since Chrome 57
+   */
+  export interface Point {
+      /** The x-coordinate of the point. */
+      x: number;
+      /** The y-coordinate of the point. */
+      y: number;
+  }
+
+  /**
+   * @since Chrome 57
+   */
+  export interface TouchCalibrationPair {
+      /** The coordinates of the display point. */
+      displayPoint: Point;
+      /** The coordinates of the touch point corresponding to the display point. */
+      touchPoint: Point;
+  }
+
+  /**
+   * @since Chrome 52
+   */
+  export interface DisplayMode {
+      /** The display mode width in device independent (user visible) pixels. */
+      width: number;
+
+      /** The display mode height in device independent (user visible) pixels. */
+      height: number;
+
+      /** The display mode width in native pixels. */
+      widthInNativePixels: number;
+
+      /** The display mode height in native pixels. */
+      heightInNativePixels: number;
+
+      /**
+       * @deprecated Deprecated since Chrome 70. Use `displayZoomFactor`
+       * @description The display mode UI scale factor.
+       **/
+      uiScale: number;
+
+      /** The display mode device scale factor. */
+      deviceScaleFactor: number;
+
+      /**
+       * The display mode refresh rate in hertz.
+       * @since Chrome 67
+       */
+      refreshRate: number;
+
+      /** True if the mode is the display's native mode. */
+      isNative: boolean;
+
+      /** True if the display mode is currently selected. */
+      isSelected: boolean;
+  }
+
+  /**
+   * @since Chrome 53
+   */
+  export interface DisplayLayout {
+
+      /** The unique identifier of the display. */
+      id: string;
+      /** The unique identifier of the parent display. Empty if this is the root. */
+      parentId: string;
+      /**
+       * The layout position of this display relative to the parent.
+       * This will be ignored for the root.
+       * @see enum
+       */
+      position: ToStringLiteral<typeof DisplayPosition>;
```

Comment:
This is not worth a type alias if you only write it once. And you can write it more simply as

```ts
position: typeof DisplayPosition[keyof typeof DisplayPosition]
```